guide/case_study.html

```diff
@@ -1043,6 +1049,7 @@ <h2>Complete Implementation</h2>
   end application
 
   define application "Chiller"
+    # Chillers are large size ACs that other simulations may put under the commercial AC category but it should be its own application category for the case study
```

Comment:
Please break this comment into multiple lines each no longer than 100 characters long.